src/jit/codegenxarch.cpp

```diff
@@ -241,7 +241,11 @@ BasicBlock* CodeGen::genCallFinally(BasicBlock* block)
     if ((compiler->lvaPSPSym == BAD_VAR_NUM) ||
         (!compiler->compLocallocUsed && (compiler->funCurrentFunc()->funKind == FUNC_ROOT)))
     {
-        inst_RV_RV(INS_mov, REG_ARG_0, REG_SPBASE, TYP_I_IMPL);
+#ifdef UNIX_X86_ABI
```

Comment:
Agree. (The extra mov ecx, rsp would be harmless - but it is better to not have it.)